sky/shell/platform/mac/sky_window.mm

```diff
@@ -100,15 +102,10 @@ - (void)setupAndLoadDart {
 
   auto args = command_line.GetArgs();
   if (args.size() > 0) {
-    _sky_engine->RunFromFile(args[0],
-        command_line.GetSwitchValueASCII(sky::shell::switches::kPackageRoot));
-    return;
-  }
-
-  NSString *endpoint = self.skyInitialBundleURL;
-  if (endpoint.length > 0) {
-    mojo::String string(endpoint.UTF8String);
-    _sky_engine->RunFromBundle(string);
+    _sky_engine->RunFromFile(
+        args[0],  //
```

Comment:
Maybe capture the result of GetSwitchValueASCII into a local to make the line format cleanly?  That's probably better than having a stray comment.  :)